src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -694,6 +694,30 @@ class UiaTextRangeTests
                     {0, bottomRow},
                     {0, bottomRow}
                 }
+            },
+
+            MoveTest{
+                L"can move to a new row when necessary when moving forward",
+                { lastColumnIndex, 0 },
+                { lastColumnIndex, 0 },
+                5,
+                {
+                    5,
+                    {lastColumnIndex , 0 + 5},
+                    {lastColumnIndex , 0 + 5}

```

Comment:
```suggestion
                    {0, 0 + 5},
                    {0, 0 + 5}
```
This'll fix the failing test. This is the expected behavior too so that's good.